backends/p4tools/common/options.h

```diff
@@ -29,6 +34,11 @@ class AbstractP4cToolOptions : protected Util::Options {
     /// @returns a compilation context on success, std::nullopt on error.
     std::optional<ICompileContext *> process(const std::vector<const char *> &args);
 
+    // No copy constructor and no self-assignments.
+    AbstractP4cToolOptions(const AbstractP4cToolOptions &) = delete;
+
+    AbstractP4cToolOptions &operator=(const AbstractP4cToolOptions &) = delete;
```

Comment:
Is move acceptable?

Reply:
That was suggested by clang-tidy. https://clang.llvm.org/extra/clang-tidy/checks/modernize/use-equals-delete.html